src/main/java/net/minecraftforge/fmp/client/multipart/FastMSR.java

```diff
@@ -0,0 +1,52 @@
+package net.minecraftforge.fmp.client.multipart;
+
+import org.lwjgl.opengl.GL11;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.client.renderer.RenderHelper;
+import net.minecraft.client.renderer.Tessellator;
+import net.minecraft.client.renderer.VertexBuffer;
+import net.minecraft.client.renderer.texture.TextureMap;
+import net.minecraft.client.renderer.vertex.DefaultVertexFormats;
+import net.minecraftforge.client.model.animation.FastTESR;
+import net.minecraftforge.fmp.multipart.IMultipart;
+
+/**
+ * Multipart equivalent of {@link FastTESR}.
+ */
+public abstract class FastMSR<T extends IMultipart> extends MultipartSpecialRenderer<T>
+{
```

Comment:
Also, vanilla still has four items which need TESRs to render, Also, isn't the animation API effectively a FastTESR?